src/components/Match/matchColumns.jsx

```diff
@@ -51,6 +52,13 @@ export const heroTdColumn = {
   sortFn: true,
 };
 
+export const heroTdChatColumn = {
+  displayName: 'Player',
```

Comment:
This string is not localized in any of the other table headings as well. I can open as a new issue, or just fix them in here. Thoughts?